src/templates/doe_workflow_ductless_heat_pump.mdx

```diff
@@ -0,0 +1,336 @@
+            experienced temperature will often be different due to the equipment’s room
+            temperature sensor being located inside the unit which is positioned near the ceiling.
+            Since warm air rises, it will be warmer near the ceiling than at head and chest level.
+            Thus, in the heating season if the temperature on the remote is set to 72 F for example,
+            the room at head and chest level will often be colder than 72 F. And in the cooling
+            season, if the remote is set to 76 F for example, the room at head and chest level will
+            often be colder than that. To compensate for this difference, one might need to set the
+            temperature to 76 F in the heating season to achieve 72 F at head and chest level.
+            There are a growing number of ductless heat pump models with features and designs that
+            mitigate some or all these problems such as the ability to use a wall thermostat, or remotes
+            with a temperature sensor built into the remote to mitigate the temperature sensor location
+            issue of high wall-mounted form factor ductless indoor units.
+            
+            ## About Ductless Heat Pump Sizing
+            While each room in a house has heating and cooling loads, ductless heat pump units are only
+            available in sizes as small as 6000 BTU/h. Therefore, very small rooms such as bathrooms
+            should not have a dedicated ductless indoor unit but rather receive heated/cooled air from a
+            ductless heat pump unit nearby. A thru wall room to room transfer fan can be installed to help
+            ensure even temperature. Some ductless indoor units have a side port designed for ducting
+            some conditioned air to an adjacent room that is too small to have a dedicated indoor unit.
+            
+        </Collapsible>
+
+      
+        <PhotoInput id="proposed_odu_location_photo" label="Proposed ODU Location – Photo">
+            Take a photo of the proposed heat pump outdoor unit (ODU) installation location. 
+        </PhotoInput>
+
+        <Radio label="ODU will be installed on a" options={['Pad','Ground Stand','Wall Mount','Roof Mount']} path="outdoor_unit.install_location" />
+
+        <NumberInput label="Average Snow Depth (in)" min={0} path="outdoor_unit.average_snow_depth"  />
+        
+        <StringInput  label="ODU Installation Location Comments" path="comment_odu_install_location" />
+
+        ## Proposed Indoor Unit   
+        <PhotoInput id="proposed_idu_location_photo" label="Proposed IDU Location – Photo">
+            Take a photo of the proposed indoor unit (IDU) installation location, if there are more than one IDU
+            installation locations, write in the notes describing the others.
+        </PhotoInput>
+        
+        <StringInput  label="IDU Location Comments" path="comment_idu_install_location" />
```

Comment:
I think the comment string input paths start with "comment_" sometimes and "comments_" other times.  We should be consistent throughout.

Reply:
All the comment string input components are labeled as 'comment_' for consistency.